ModuleFourTasks/Task3/UserTaskService.cs

```diff
@@ -12,25 +12,29 @@ public UserTaskService(IUserDao userDao)
             _userDao = userDao;
         }
 
-        public int AddTaskForUser(int userId, UserTask task)
+        public void AddTaskForUser(int userId, UserTask task)
         {
             if (userId < 0)
-                return -1;
+            {
+                throw new ArgumentException("Invalid userId");
```

Comment:
More appropriate exception here will be `ArgumentOutOfRangeException`

Reply:
Fixed in cbd4ea3.